streams.yml

```diff
@@ -106,6 +106,17 @@ etcd_golang:
   upstream_image_base: registry.ci.openshift.org/ocp/builder:rhel-8-etcd-golang-1.16
   upstream_image: registry.ci.openshift.org/ocp/builder:rhel-8-etcd-golang-1.16
 
+# IMPORTANT: etcd has unique approval to track its own etcd version. Other repos need arch
+# approval to diverge from what kube apiserver uses for a given release.
+# https://coreos.slack.com/archives/CB95J6R4N/p1598453188186800?thread_ts=1598449075.172000&cid=CB95J6R4N
```

Comment:
i don't think this thread has been preserved, probably best to omit this line

Reply:
Good point, fixed